addons/mail/models/discuss/discuss_channel_member.py

```diff
@@ -402,11 +402,38 @@ def _get_rtc_server_info(self, rtc_session, ice_servers=None, key=None):
         json_web_token = jwt.sign(claims, key=key, ttl=60 * 60 * 8, algorithm=jwt.Algorithm.HS256)  # 8 hours
         return {"url": sfu_server_url, "channelUUID": sfu_channel_uuid, "jsonWebToken": json_web_token}
 
-    def _rtc_leave_call(self):
+    def _rtc_leave_call(self, device_endpoint=None):
         self.ensure_one()
         if self.rtc_session_ids:
             self.rtc_session_ids.unlink()
-        else:
+            if not device_endpoint or not self.partner_id:
+                return
+            devices, private_key, public_key = self.channel_id._get_web_push_parameters([self.partner_id.id], endpoint=device_endpoint)
+            if not devices:
+                return
+            self.channel_id._push_web_notification(devices, private_key, public_key, payload=truncate_payload({
```

Comment:
Are we still sending a notification for the user to click it? Didn't we settle on a direct recover without any user action?

Reply:
I was waiting that we settle on an approach before converting, will do next